docs/source/contributing.rst

```diff
@@ -61,4 +61,31 @@ Finalizing a Pull Request
 
 Once the PR is submitted, we will look through it and request any changes necessary before merging it into the main branch. You can make those changes just like any other edits on your fork. Then when you push them, they will be joined in to the PR automatically and any unit tests will run again.
 
+Black Formatting Exceptions for Equations
+-----------------------------------------
+
+In the "caustics" project, we utilize the Black code formatter to ensure consistent and readable code. However, there are instances where the automatic formatting performed by Black may not align with the desired formatting for equations within the code.
```

Comment:
Perhaps caustics should be bold (**caustics**) or highlighted (`caustics`), or written normally, since we don't specially highlight caustics elsewhere in the docs

Reply:
Thanks for the feedback. I just made that change for you.

> Perhaps caustics should be bold (**caustics**) or highlighted (`caustics`), or written normally, since we don't specially highlight caustics elsewhere in the docs

Thanks for the feedback. I just made that change for you.